shab.nix

```diff
@@ -0,0 +1,25 @@
+{ bash, stdenv, lib, ... }:
+
+stdenv.mkDerivation {
+  name = "shab";
+  version = "1";
```

Comment:
should that go in the name?

Reply:
We need an official versioning policy and processes to change the versioning policy and the version of the versioning policy. Wait...